docs/hcp-upload.md

```diff
@@ -0,0 +1,24 @@
+# HCP Upload
+
+For commands that HCP upload is enabled you will need to go through the following configuration steps within your environment in order to upload.
+
+## Creating your Service Principals
+
+You wil need to log into the HCP web portal to create your service principals. You can follow the HCP docs in order to create your [Project Service Principals](git@github.com:hashicorp-guides/vault-radar-demo.git). You will need to create the Service Principals at Admin level, more information about this level contained in [these](git@github.com:hashicorp-guides/vault-radar-demo.git) HCP docs.
```

Comment:
It looks like both the links are pointing to `vault-radar` git url, is that expected?